owncloudApp/src/main/res/layout/account_item.xml

```diff
@@ -15,107 +15,146 @@
+            android:layout_marginStart="8dp"
+            android:ellipsize="middle"
+            android:text="@string/placeholder_sentence"
+            android:textColor="@color/textColor"
+            android:textSize="14sp"
+            android:lines="1"
+            app:layout_constraintEnd_toStartOf="@+id/clean_account_local_storage_button"
+            app:layout_constraintStart_toEndOf="@id/ticker"
+            app:layout_constraintTop_toBottomOf="@id/name" />
+
+        <ImageView
+            android:id="@+id/clean_account_local_storage_button"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_vertical"
+            android:background="?android:attr/selectableItemBackground"
+            android:clickable="true"
+            android:paddingStart="@dimen/standard_half_padding"
+            android:paddingTop="@dimen/standard_padding"
+            android:paddingEnd="@dimen/standard_half_padding"
+            android:paddingBottom="@dimen/standard_padding"
+            android:src="@drawable/ic_clean_account"
+            android:contentDescription="@string/content_description_clean_account_storage"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toStartOf="@+id/removeButton"
+            app:layout_constraintTop_toTopOf="parent"
+            app:tint="@color/black" />
+
+        <ImageView
+            android:id="@+id/removeButton"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_vertical"
+            android:background="?android:attr/selectableItemBackground"
+            android:clickable="true"
+            android:paddingStart="@dimen/standard_half_padding"
+            android:paddingTop="@dimen/standard_padding"
+            android:paddingEnd="@dimen/standard_padding"
+            android:paddingBottom="@dimen/standard_padding"
+            android:src="@drawable/ic_action_delete_grey"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            android:contentDescription="@string/content_description_remove_account"
+            app:tint="@color/black" />
+
+    </androidx.constraintlayout.widget.ConstraintLayout>
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="20dp"
+        android:layout_marginStart="@dimen/standard_margin"
+        android:layout_marginEnd="@dimen/standard_margin"
+        android:layout_marginTop="@dimen/standard_half_margin">
+
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/used_quota_title_manage_accounts"
+            android:layout_marginEnd="10dp"
+            android:textSize="14sp"
```

Comment:
Done. I've kept `125dp` in progress bar because it's not a standard dimension and it's necessary in this case to avoid problems with the quota text length.